Topics/Software_Tools/Markdown/Markdown.md

```diff
@@ -0,0 +1,68 @@
+
+<img width="113" alt="linebreak1" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/4d803c2a-8d30-4e72-bc08-869835099faf">
+<img width="108" alt="linebreak2" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/52a8d3ea-bede-4a3e-b687-d2d77fe3f551">
+
+### Text Emphasis (Bold & Italics)    
+#### Bold
+To bold a word, phrase, or block of text, either surround it with two astericks (*) or two underscores (_) on each side. An example is displayed below. 
+
+<img width="234" alt="bold2" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/0318e78a-45c3-4d4d-8602-0a0ead5d9f70">
+<img width="213" alt="bold1" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/b953bf11-aaad-471f-a722-1ed198348c92">
+
+To bold certain characters within a word, surround the characters with double astericks on each side. An example is displayed below.
+<img width="107" alt="bold3" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/75686194-63f8-4f55-b14a-52d63f0d110c">
+<img width="96" alt="bold4" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/1a42367b-8ffb-4caa-8467-5eab33d10991">
+
+#### Italics 
+Italicizing text is done in similar fashion to bolding text. To italicize words or phases, surround the text with a single asterick (*) or underscore (_) on each side. To italicize certain characters within a word, surround the characters with a single asterick on each side. An example is shown below.  
+<img width="182" alt="italicize1" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/b4c369c8-cee5-4835-935c-d96db4f71d71">
+<img width="162" alt="italicize2" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/00e0e340-c5a3-4be9-9de9-0b69113583c6">
+
+
+#### Bold and Italics
+To bolden and italicize text at the same time, surround the text with a pair of any of the following options displayed below, where the first two options are three astericks and three underscores. To bolden and italicize characters within a word, surround the characters with three astericks. An example is shown below.   
+<img width="183" alt="both" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/dd07d4a2-2cc7-4473-a913-3b8e0a9e4f6b">
+<img width="142" alt="both2" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/74106672/2f6c3381-9cd6-48dd-8dc0-134636decf38">
+
+
+### Lists
```

Comment:
Great detail! You could also add how to create nested lists?

Reply:
Greaet suggestion! I added a section regarding this topic including a side by side example image below.